prog/test/vm.rb

```diff
@@ -22,7 +22,7 @@ class Prog::Test::Vm < Prog::Test::Base
   end
 
   label def install_packages
-    if vm.boot_image.start_with?("ubuntu")
+    if /(ubuntu)|(debian)/.match?(vm.boot_image)
```

Comment:
```suggestion
    if /ubuntu|debian/.match?(vm.boot_image)
```

You aren't using the capture groups (not that it would matter as you are using `match?`).